src/ECR.jl

```diff
@@ -17,16 +21,16 @@ The AWS `config` keyword parameter is optional (will use the default if it's not
 """
 get_login
 
-function get_login(registry_id::AbstractString=""; config::AWSConfig=aws_config())
+function get_login(registry_id::AbstractString=""; config::AWSConfig=global_aws_config())
     # Note: Although `get_authorization_token` can take multiple registry IDs at once it
     # will only return a "proxyEndpoint" for the first registry. Additionally, the
     # `aws ecr get-login` command returns a `docker login` command for each registry ID
     # passed in. Because of these factors we'll do our processing on a single registry.
 
     response = if !isempty(registry_id)
-        @mock get_authorization_token(config; registryIds=[registry_id])
+        @mock get_authorization_token(config, Dict("registryIds" => [registry_id]))
     else
-        @mock get_authorization_token(config)
+        @mock get_authorization_token(config, Dict{String,Any}())
```

Comment:
```suggestion
        @mock get_authorization_token(config)
```

Then split the patch? I don't know how Mocking patches with optional arguments work.

Reply:
Idk maybe Mocking makes this too hard

Reply:
I'm gonna try some witchcraft here. Will update in a bit with what happens.